contracts/Cross-chain/OmnichainGovernanceExecutor.sol

```diff
@@ -118,7 +118,7 @@ contract OmnichainGovernanceExecutor is ReentrancyGuard, BaseOmnichainController
      * @notice Add timelocks to the ProposalTimelocks mapping.
      * @param timelocks_ Array of addresses of all 3 timelocks.
      * @custom:access Only owner.
-     * @custom:event Emits TimelockAdded with all 3 timelocks.
+     * @custom:event Emits TimelockAdded with old and new timelock and route type.
      */
     function addTimelocks(ITimelock[] memory timelocks_) external onlyOwner {
```

Comment:
Related to VPB-04, instead of hardcoding `ProposalType.CRITICAL`, we could calculate the number of proposal types dynamically:
```
uint8 totalProposalTypes = uint8(type(ProposalType).max) + 1;
```

Reply:
https://github.com/VenusProtocol/governance-contracts/pull/59/commits/2c94bd93f3559aa50688abcad257dac9d5f9c645